cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -153,6 +153,38 @@
         bottom: unit( @grid_gutter-width / 2 / @play-btn-icon-size, em );
     }
 
+    &__center,
+    &__bottom-right {
+        &:focus-visible {
+            outline: none;
+
+            &:before,
+            &::after { 
```

Comment:
You have an after here with a double colon `::after`. Technically double colons is to spec, but it is not supported in older browsers. See https://css-tricks.com/to-double-colon-or-not-do-double-colon/